pkg/drivers/kic/oci/context.go.bak

```diff
@@ -0,0 +1,288 @@
+/*
```

Comment:
@einyx please rebaase and remove the un-intended files so I could review it once it is ready